conf.yaml

```diff
@@ -853,8 +854,9 @@ contents:
             prefix:     en/cloud-enterprise
             tags:       CloudEnterprise/Reference
             subject:    ECE
-            current:    ms-69
+            current:    ms-70
             branches:
+              - ms-70: 3.1.0
```

Comment:
Should this be `3.1.0` or `3.1` (to match the existing entries)?

Reply:
Likely yes. I'll fix that.